content/exchange-guide.md

```diff
@@ -0,0 +1,575 @@
+
+* Withdraw XRP _from_ Alpha Exchange _to_ the RCL
+
+* Trade XRP with other currencies, such as BTC, USD, amongst others
+
+## Prerequisites for Supporting XRP
+
+To support XRP, Alpha Exchange must:
+
+* Create and maintain new [accounts](#accounts)
+
+* Create and maintain [balance sheets](#balance-sheets)
+
+See also:
+
+* [Gateway Compliance](https://ripple.com/build/gateway-guide/#gateway-compliance) — Gateways and exchanges are different, but exchanges should also ensure that they are complying with local regulations and reporting to the appropriate agencies.
+
+* [Requirements for Sending to RCL](https://ripple.com/build/gateway-guide/#requirements-for-sending-to-rcl)
+
+* [Requirements for Receiving from RCL](https://ripple.com/build/gateway-guide/#requirements-for-receiving-from-rcl)
+
+* [Gateway Precautions](https://ripple.com/build/gateway-guide/#precautions)
+
+### Accounts
+
+XRP is held in *accounts* (sometimes referred to as *wallets* ) on the Ripple Consensus Ledger (RCL). Accounts on the RCL are different than accounts on other blockchain ledgers, such as Bitcoin, where accounts incur little to no overhead. To submit transactions (for example, [OfferCreate](https://ripple.com/build/transactions/#offercreate) and others used for trading), RCL accounts require XRP [reserves](https://ripple.com/build/reserves/) to protect the ledger against spam and malicious usage. On other blockchains, balances are derived from the previous block. On the RCL, [account objects](https://ripple.com/build/ledger-format/#accountroot) describe several other properties of the account in addition to balances, so accounts are represented in each ledger and can never be destroyed or removed. Exchanges do not need to create accounts for each customer that holds XRP; they can store all of their customers’ XRP in just a few RCL accounts. For more information about RCL accounts, see the [Accounts](https://ripple.com/build/accounts/) article.
+
+To comply with Ripple's recommend best practices, Alpha Exchange should create at least two new [accounts](https://ripple.com/build/accounts/) on the RCL. To minimize the risks associated with a compromised secret key, Ripple recommends creating [_issuing_, _operational_, and _standby_ accounts](https://ripple.com/build/issuing-operational-addresses/) (these are sometimes referred to, respectively, as cold, hot, and warm wallets). The operational/standby/issuing model is intended to balance security and convenience. Exchanges listing XRP should create the following accounts:
+
+* An [_issuing_ account](https://ripple.com/build/issuing-operational-addresses/#issuing-address) to securely hold the majority of XRP and customers' funds. To provide optimal security, this account should be offline.
+
+    For more information about the possible consequences of a compromised issuing account, see [Issuing Account Compromise](https://ripple.com/build/issuing-operational-addresses/#issuing-address-compromise).
```

Comment:
This link is one of the cases where exchanges and gateways are really different. The consequences of a compromised cold wallet for an XRP-only exchange are:

- The malicious actor gets full access to all XRP in the cold wallet
- If the master key is compromised, the malicious actor can irrevocably take control of the cold wallet forever (by disabling the master key and setting a new regular key or signer list), giving the malicious actor control over all _future_ XRP received by the cold wallet too
    - If this happens, the exchange has to make a new cold wallet address and tell its customers the new address.
- If only the regular key/signer list are compromised, the exchange can regain control of the cold wallet, but some actions cannot be easily undone.
    - The malicious actor could issue currency in the RCL using the cold wallet, but most people should not value that currency at anything unless the exchange explicitly said it was going to be a gateway too, so this can be ignored
    - Any settings an attacker could set/unset without the master key can be just as easily changed back, except for `asfRequireAuth` which only relates to issuing currency anyway.